ios-showcase-template.xcodeproj/project.pbxproj

```diff
@@ -897,15 +899,17 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = "ios-showcase-template/ios-showcase-template.entitlements";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = GHPBX39444;
 				INFOPLIST_FILE = "ios-showcase-template/Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = "org.aerogear.ios-showcase-template";
+				PRODUCT_BUNDLE_IDENTIFIER = org.aerogear.ShowcaseTemplate;
```

Comment:
We should probably change also this [1] and maybe [2] (not sure if it is used anywhere).

[1] https://github.com/aerogear/ios-showcase-template/blob/83f5a26c615f1dc4b27abf5dc978a9434b02c842/ios-showcase-template/di/AppComponents.swift#L35
[2] https://github.com/aerogear/ios-showcase-template/blob/4f3ac372d06ca512d54fb1be86bc412ee29e6c37/ios-showcase-template/Info.plist#L26